superset/assets/javascripts/explore/stores/controls.jsx

```diff
@@ -1317,6 +1382,20 @@ export const controls = {
     'Between 0 and 1.'),
   },
 
+  viewport: {
+    type: 'ViewportControl',
+    label: t('Viewport'),
+    renderTrigger: true,
+    description: t('Parameters related to the view and perspective on the map'),
+    default: {
+      longitude: 6.85236157047845,
```

Comment:
These looks like the same numbers that are hardcoded as defaults, do we need them in both locations?

Reply:
moving as `defaultViewport` in utils/geo.js